src/BreadcrumbItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import SafeAnchor from './SafeAnchor';
+
+const BreadcrumbItem = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    linkId: React.PropTypes.string,
+    href: React.PropTypes.string,
+    title: React.PropTypes.node,
+    target: React.PropTypes.string
+  },
+
+  render() {
+    const {
+      href,
+      title,
+      target,
+      linkId,
+      children,
+      ...props } = this.props;
+    const active = (href == null);
```

Comment:
I think spread the props onto the anchor is a better idea since we don't assign props to `li` in most cases. But what if an `anchor` is not rendered in an active component? Do I just ignore the props which are assigned to an active component?